src/lib/utility/Dropdown.svelte

```diff
@@ -24,19 +27,20 @@
 </script>
 
 <svelte:window on:click={(e) => {
-    if (open && !button.contains(e.target)) {
+    if (open && getState() && !button.contains(e.target)) {
         open = false;
+        onClick();
     }
 }}></svelte:window>
 
 <!-- svelte-ignore a11y-click-events-have-key-events -->
-<div bind:this={button} class="dropdown-button" class:open on:click={() => {open = !open}}>
+<div bind:this={button} class="dropdown-button" class:open on:click={() => {onClick(); open=true}} on:mouseenter={() => open = true} on:mouseleave={() => open=false}>
```

Comment:
`on:click={() => {onClick(); open=true}}`
This looks weird since `onClick()` is just `open = !open`, and then you set open to true anyways.

Reply:
thats the default value of onClick, since you have used the Dropdown component outside the menubar menu group too (eg. the settings), but yes, it seems obsolete, so it can be removed!